js/display-pictures.ts

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import { Description } from './mock.ts';
+
+const pictureList = document.querySelector('.pictures');
+const pictureTemplate = document.querySelector('#picture')?.querySelector('.picture');
```

Comment:
Забыл контент. Через знак вопроса, можно сделать код надежнее
```suggestion
const pictureTemplate = document.querySelector<HTMLTemplateElement>('#picture')?.content.querySelector<HTMLAnchorElement>('.picture');
```